brainscore_language/models/lm1b/data_utils.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
if this is copied from TensorFlow, I suggest indicating the github source (including the commit, i.e. press `y` in github before copying the url)

Reply:
Resolved in the latest commit